# Use ovs_accel_flow_find_no_ct function for flow lookup, which only uses info
# ECM is actually aware of to find flow.
--- a/ovsmgr_dp.c
+++ b/ovsmgr_dp.c
@@ -1675,7 +1675,7 @@ enum ovsmgr_flow_status ovsmgr_dp_flow_i
 	 */
 	ofi->dscp = OVSMGR_INVALID_DSCP;
 
-	sf = ovs_accel_flow_find(nod->dp, &key);
+	sf = ovs_accel_flow_find_no_ct(nod->dp, &key, true);
 	if (!sf) {
 		if (!flow->is_routed) {
 			ovsmgr_warn("%px:Couldn't find flow rule \n", flow);
